.travis.yml

```diff
@@ -4,6 +4,15 @@ node_js:
   - "0.11"
   - "0.10"
   - "iojs"
+install:
```

Comment:
Very cool. I'm familiar with the job that travis does but the configuration of travis is awesome.